core/audits/network-requests.js

```diff
@@ -64,8 +64,11 @@ class NetworkRequests extends Audit {
       return {
         url: UrlUtils.elideDataURI(record.url),
         protocol: record.protocol,
-        startTime: normalizeTime(record.startTime),
-        endTime: normalizeTime(record.endTime),
+        rendererStartTime: normalizeTime(record.rendererStartTime),
+        networkRequestTime: normalizeTime(record.networkRequestTime),
+        responseHeadersEndTime: normalizeTime(record.responseHeadersEndTime),
+        networkEndTime: normalizeTime(record.networkEndTime),
+        rendererEndTime: normalizeTime(record.rendererEndTime),
```

Comment:
adding the extra new properties could possibly wait until all the other audits are switched over to using them (where appropriate). That would keep those changes separate from the renames.

Reply:
Kom bedre å bare ta det litt av Kombef er litt for tynn Henrik